### PR TITLE
Add action text field

### PR DESCRIPTION
**Pull request by composerinteralia**
Some of this was taken from https://github.com/thoughtbot/administrate/wiki/Field:-RichTextAreaField

### CODE REVIEW

**Comment by composerinteralia** [comment]
To see this in action:

```sh
set -e

# Install all the things
rails new test_app
cd test_app
echo 'gem "administrate", github: "thoughtbot/administrate", branch: "add-action-text-field"' >> Gemfile
echo "gem 'image_processing', '~> 1.2'" >> Gemfile
bundle install
rails action_text:install
yarn install

# Scaffold
rails generate scaffold Article
sed -i "" '2i\
  has_rich_text :content
' app/models/article.rb
rails db:migrate

# Install administrate
rails generate administrate:install
echo '//= link administrate/application.css
//= link administrate/application.js' >> app/assets/config/manifest.js

# Don't mind me...
cd `bundle show administrate`
bundle install
yarn install
cd -

rails server
```

File: app/assets/stylesheets/administrate/library/action-text.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    }
+  }
+
+  action-text-attachment {
+    .attachment {
+      padding: 0 !important;
+      max-width: 100% !important;
+    }
+  }
+}
```
Comment (jsilasbailey) [inline app/assets/stylesheets/administrate/library/action-text.scss]: This makes me sad that we have to do this, but 🤷‍♂

File: app/views/fields/action_text/_index.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+<%= field.to_s %>
```
Comment (jsilasbailey) [inline app/views/fields/action_text/_index.html.erb]: Maybe this should not display the all the content on the index view by default? Or have some option around this

**Comment by pablobm** [comment]
Closing due to lack of activity.

**Pull request closed without merging**